phpunit.relay.xml

```diff
@@ -25,7 +25,6 @@
             <group>realm-stack</group>
             <group>ext-curl</group>
             <group>ext-phpiredis</group>
-            <group>cluster</group>
```

Comment:
@tillkruss did you mean this?

Reply:
Yes, let's see if the tests run.

Reply:
@tillkruss I have run following locally with all tests passing on PHP 8.1 and Redis 7.2:

- `vendor/bin/phpunit` with relay extension disabled
- `vendor/bin/phpunit -c phpunit.relay.xml` with relay extension enabled

I found [one failing test which i fixed](https://github.com/predis/predis/blob/de89d9d4ec8e6c9e5bc83a72f54da9ecf942cf6e/tests/Predis/Command/Redis/CLUSTER_Test.php#L99), but that one was failing even before my changes (i tested that on the `v2.x` branch).

Can you please run the checks to see if they pass this time?

Reply:
I’ve changed the permissions, so they should run automatically now.

Reply:
@tillkruss Turns out you cant remove `cluster` from exclusions in `phpunit.relay.xml` because `predis` tests run without redis cluster. There is a separate job `predis-cluster` which spins up a redis cluster, but that one defines the group explicitly to be `cluster`. I have therefore instead opted for expanding `predis-cluster` job to include a phpunit run with `phpunit.relay.xml`.